packages/design-system/src/components/primitives/arrow-focus.tsx

```diff
@@ -0,0 +1,162 @@
+import type { KeyboardEvent } from "react";
+
+const focusableSelector = (extra: string) =>
```

Comment:
I am surprised you ended up not using react-aria, I thought we decided to use it?

Reply:
It doesn't do what we need: https://discord.com/channels/955905230107738152/973300187567095858/1085820876060106812

Reply:
Talked in DM: I misunderstood what FocusScope is, need to have another look. Maybe we can use it.